modules/claims_api/app/controllers/claims_api/v2/veteran_identifier_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'evss/error_middleware'
+
+module ClaimsApi
+  module V2
+    class VeteranIdentifierController < ApplicationController
+      skip_before_action :authenticate, only: %i[show]
+      ICN_FOR_TEST_USER = '1012667145V762142'
```

Comment:
does it make more sense to have this value be randomly generated each call? i can sort of see benefits either way i guess.

Reply:
eh, we had talked about that yesterday during standup and Mark said he'd rather we just return the ICN of a test user.